### ISSUE TITLE
No author displayed in Elsevier and Springer journals

### ISSUE THREAD

**Issue by MPIKGLibrary** [bibliocoll/JournalTouch#92]
Opening a journal issue published by Elsevier or Springer (e.g. Advances in Colloid and Interface Science) we get the titles of the published articles but no authors. Does anyone have an idea how this can be dealt with?

**Comment by MPIKGLibrary** [comment]
We use version 4.3

**Comment by MPIKGLibrary** [comment]
We contacted JournalTOCs and they were so nice to change the metadata of the Springer Journals. Hopefully the Elsevier Journals will be dealt with as well. Obviously the question, whether Autors are displayed in Journal Touch or not, depends on the metadata provided by JournalTOCs.

**Comment by tzeumer** [comment]
Yes, since JournalTouch is very much depends on JournalTocs (CrossRef might sometimes serve as fallback) the publisher's data is very important. JournalToc published an interesting blog post covering this topic (for publishers): https://www.journaltocs.ac.uk/API/blog/?tag=rss-best-practices.

In our experience especially "single" Open Access journals seem to have shortcomings in this regard. But JournalToc is the only (free) source of it's kind, but I don't think incoming data are curated except for some field mappings.
